Frontend/src/components/rankings/ResultsTableTeams.vue

```diff
@@ -69,18 +69,13 @@
   </section>
 </template>
 
-<script>
-export default {
-  name: "ResultsTable",
-  props: {
-    results: {
-      type: Array,
-      required: true,
-    },
+<script setup>
+const props = defineProps({
+  results: {
+    type: Array,
+    required: true,
   },
-  data() {
-    return { maxFlights: 3 };
-  },
-};
+});
+const maxFlights = 3;
```

Comment:
Können wir solche Sachen bitte zentraler definieren.

Ich habe z.B. für das Backend einen Unterordner /constants angelegt. Idee wäre sowas in Zukunft auch besser zu synchronisieren.

Ich könnte z.B. auch mit /clubs, bzw. /teams ein maxFlights zurückliefern.

Reply:
ja schick das doch gerne über die api mit.
Habe das bei ResultsTable nicht fix, sondern as prop. War mir noch nicht sicher wie es am schlauesten ist.
Aber wenn es nun von der API kommt ist doch perfekt.

Reply:
Vielleicht aber nicht als maxflights sondern was aussagekräftiges wie numberOfScoredFlights